draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1218,568 +1044,686 @@ uCDN-A                         dCDN-B                         dCDN-C
+	the IANA "CDNI CI/T Trigger Specs" registry:
    </t>
+      
+   <texttable style="full"><ttcol align="left"> JSON String</ttcol>
+   <ttcol align="left"> Description</ttcol>
+   <c>urls</c>
+   <c>Allowing the specification of trigger targets via URLs.</c>
+   <c>ccids</c>
+   <c>Allowing the specification of trigger targets via CCIDs content grouping, as defined in section 4.2.8 <xref target="RFC8006"/>.</c>
+   <c>uri-pattern-match</c>
+   <c>Allowing the specification of trigger targets via <xref target="RFC3986"/> URI patterns.</c>
+   <c>uri-regex-match</c>
+   <c>Allowing the specification of trigger targets via regexes matching their URI, as defined in <xref target="uri-regex-match-spec"/>.</c>
+   <c>content-objectlist</c>
+   <c>Allowing the specification of trigger targets via an object list and an object list type.</c>
+   </texttable>
 
-   <t hangText="Name: coll-label">
-   <vspace blankLines="1"/>
-   Description: Array of all <xref target="filtered-collection-label-link"/> objects.
-   <vspace blankLines="1"/>
-   Value: An array of JSON-encoded FilteredCollectionLabelLink objects.
-   <vspace blankLines="1"/>
-   Mandatory: Mandatory in the collection of all Trigger Status Resources, if dCDN
-         implements the filtered collections. Otherwise, optional.
+   <t>
+        Each trigger usually refers to the targets by the target URLs, using a "urls" trigger spec object or
+        some aggregating spec such as the "url-regex-match".
+        If content URLs are transformed by an intermediate CDN in a cascade,
+        that intermediate CDN MUST similarly transform URLs in triggers it passes to its dCDNs.</t>
+   <t>
+        When processing a trigger, CDNs MUST ignore the URL scheme (HTTP or HTTPS) in comparing URLs.
+        For example, for a CI/T "invalidate" or "purge" action, content MUST be invalidated or
+        purged regardless of the protocol clients used to request it.
    </t>
 
-   <t hangText="Name: all-triggers">
-   <vspace blankLines="1"/>
-   Description: Array of all Trigger Status objects in the collection.
-   Should be returned only when an extended trigger collection view is requested as described in 
-   <xref target="polling-collections-extended"/>.
-   <vspace blankLines="1"/>
-   Value: An array of JSON-encoded Trigger Status objects.
-   <vspace blankLines="1"/>
-   Mandatory: No. The "all-triggers" SHOULD only be used by dCDN that supports and advertises 
-   the appropriate extended status for trigger collections (see <xref target="extended-status-capability-object"/> for details).
+   <section anchor="generic-spec-object" title="Generic Spec Object">
+   <t>
+	A trigger resource, as defined in <xref target="ci-trigger-resource"/>, includes an array of trigger spec objects. 
+	Each trigger spec object contains properties that are used as trigger target selection directives for dCDN when processing the trigger. 
```

Comment:
Punctuation:  Should be, "processing the trigger, e.g., content URLs"

Reply:
added commas after all occurrences of i.e. and e.g. in the document